SlipstreamIgnoreFileLoader.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace CommBank.Api.Provisioning.Services.Source
+{
+    /// <summary>
+    /// Provides the ability to load .ssignore file content, if one is present in the scaffold.
+    /// </summary>
+    public interface ISlipstreamIgnoreFileLoader
+    {
+        /// <summary>
+        /// Loads .ssignore content as individual rows.
+        /// </summary>
+        /// <param name="path">The path to folder where .ssignore file is located.</param>
+        /// <returns>Non empty entries from the .ssignore file; <see langword="null"/> if the file can't be loaded.</returns>
+        List<string> LoadEntries(string path);
+    }
+
+    /// <summary>
+    /// Loads .ssignore file content, if one is present in the scaffold.
+    /// </summary>
+    public sealed class SlipstreamIgnoreFileLoader : ISlipstreamIgnoreFileLoader
+    {
+        public const string IgnoreFileName = ".ssignore";
```

Comment:
1. if it need in other places - 1. we can move to common const file
2. if it is for SlipStream - make it is private 
3. if it in specific to Get, move inside get